src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.head {
+  width: 100%;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.inner__container {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  position: relative;
+  bottom: 1px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px;
+  margin-right: 0;
+  margin-left: 50px;
+}
+
+.nav__link {
+  display: flex;
+  text-transform: uppercase;
+  padding: 0;
+  margin: 0;
+  margin-right: 50px;
+  list-style: none;
+}
+
+.refer:link {
```

Comment:
```suggestion
.refer {
```

Reply:
I fixed it, but could you explain me why it incorrectly, i did it according to lesson and developer.mozilla.org
![image](https://user-images.githubusercontent.com/70288388/195716605-9ad239ec-f492-4b71-882b-d7032dfccdf7.png)
![image](https://user-images.githubusercontent.com/70288388/195716668-ca509810-48d7-4e49-af6b-ad2773d03e55.png)